infra/docker-compose/docker-compose.online.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  online-serving:
+    image: ${FEAST_SERVING_IMAGE}:${FEAST_VERSION}
+    volumes:
+      - ./serving/${FEAST_ONLINE_SERVING_CONFIG}:/etc/feast/application.yml
+      - ./serving/${FEAST_ONLINE_STORE_CONFIG}:/etc/feast/store.yml
+    depends_on:
+      - core
+      - redis
```

Comment:
This `depends_on` dependency should be removed as `core`  and `redis` are in separate docker-compose file.
`depends_on` only works for services on the same docker-compose file.

Reply:
Are you sure about this? Doesn't compose merge the files first?

Reply:
I see, if we are going for this `docker-compose` command, this would not be a problem.
`docker-compose -f docker-compose.yml -f docker-compose.online.yml`